app/workers/travis_branch_monitor.rb

```diff
@@ -0,0 +1,104 @@
+require 'travis'
+
+class TravisBranchMonitor
+  include Sidekiq::Worker
+  sidekiq_options :queue => :miq_bot_glacial, :retry => false
```

Comment:
Should this be on glacial?  We generally only put the long running stuff on there, and this one seems pretty short.  @bdunne Thoughts?

Reply:
Totally was a "copy-pasta" here, so I have no opinions personally.

As far as being "quick", it depends on how many repos we are working with, but assuming the Travis API isn't haven't issues (which I have noticed while testing with just a single repo, FYI), it shouldn't take more than a minute to run this with the number of repos we probably would enable, but I didn't have a large sample size to test with.

Reply:
Done.